dapps/templates/demo/app/components/whisper.js

```diff
@@ -40,7 +42,7 @@ class Whisper extends React.Component {
     e.preventDefault();
 
     const subscribedChannels = this.state.subscribedChannels;
-    subscribedChannels.push(<span>Subscribed to <b>{this.state.listenTo}</b>. Now try sending a message</span>);
+    subscribedChannels.push(this.state.listenTo);
```

Comment:
I've also moved the template code into the render function and have the state control what's being rendered.